extras/docker/centos/cccp.yaml

```diff
@@ -0,0 +1,11 @@
+# This file is required for the purpose of building a Docker container on CCCP.
```

Comment:
What is CCCP? I see this and think Soviet Union! :-)

How does this stuff work, could we add a short paragraph to the readme that explains this file?

Reply:
This file is a requirement from the registry. Details are in the README at https://github.com/CentOS/container-index